src/Resources/public/admin/js/updateProductSlug.js

```diff
@@ -0,0 +1,44 @@
+$(function () {
+  let $body = $('body');
+
+  hideAutomaticRedirectionInput();
+  $body.on('click', '.toggle-product-slug-modification', function () {
```

Comment:
The field with the class `toggle-product-slug-modification` is present in the DOM from the very beginning, right? If that's the case I don't think we should add the listener to the `body`, but instead the actual element.

Reply:
I believe it might change from one adaptation to another, and binding it to the body allows to load in ajax and so on. It's a habit I've taken in order not to struggle whenever it changes.
Indeed it can overload the DOM, but not that much to be honest.

Reply:
Okay, let's keep it this way :)